.travis.yml

```diff
@@ -37,13 +41,16 @@ before_script:
  - "if [ \"$BEHAT_TEST\" = \"\" ]; then php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss; fi"
  - "if [ \"$BEHAT_TEST\" = \"1\" ]; then php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/behat-extension; fi"
  - cd ~/builds/ss
+ - "if [ \"$CMS_TEST\" = \"1\" ]; then composer require --prefer-dist \"silverstripe/cms:$CORE_RELEASE.x-dev\"; fi"
```

Comment:
If CMS_TEST = 1, it's too late to install here. You need to include silverstripe/cms at the `~/travis-support/travis_setup.php` step, since that script removes the Page.php if it detects CMS isn't installed.